MicroProfile-Rest-Client/pom.xml

```
@@ -54,8 +54,8 @@
     <name>MicroProfile Rest Client TCK Runner - Parent</name>
 
     <properties>
-        <microprofile.rest.client.tck.version>3.0.1</microprofile.rest.client.tck.version>
-        <microprofile.rest.client.api.version>3.0.1</microprofile.rest.client.api.version>
+        <microprofile.rest.client.tck.version>4.0</microprofile.rest.client.tck.version>
+        <microprofile.rest.client.api.version>4.0</microprofile.rest.client.api.version>
```

```suggestion
        <microprofile.rest.client.tck.version>3.0.1</microprofile.rest.client.tck.version>
        <microprofile.rest.client.api.version>3.0.1</microprofile.rest.client.api.version>
```
This is for MicroProfile 7 which we don't support